src/System.Private.Xml/src/System.Private.Xml.csproj

```diff
@@ -733,97 +733,8 @@
-    <Compile Include="Workarounds\CodeDom\CodeTypeDeclaration.cs" />
-    <Compile Include="Workarounds\CodeDom\CodeTypeDeclarationCollection.cs" />
-    <Compile Include="Workarounds\CodeDom\CodeTypeMember.cs" />
-    <Compile Include="Workarounds\CodeDom\CodeTypeMemberCollection.cs" />
-    <Compile Include="Workarounds\CodeDom\CodeTypeOfExpression.cs" />
-    <Compile Include="Workarounds\CodeDom\CodeTypeParameter.cs" />
-    <Compile Include="Workarounds\CodeDom\CodeTypeParameterCollection.cs" />
-    <Compile Include="Workarounds\CodeDom\CodeTypeReference.cs" />
-    <Compile Include="Workarounds\CodeDom\CodeTypeReferenceCollection.cs" />
-    <Compile Include="Workarounds\CodeDom\CodeTypeReferenceExpression.cs" />
-    <Compile Include="Workarounds\CodeDom\CodeVariableDeclarationStatement.cs" />
-    <Compile Include="Workarounds\CodeDom\CodeVariableReferenceExpression.cs" />
-    <Compile Include="Workarounds\CodeDom\Compiler\CodeDomConfigurationHandler.cs" />
-    <Compile Include="Workarounds\CodeDom\Compiler\CodeDomProvider.cs" />
-    <Compile Include="Workarounds\CodeDom\Compiler\CodeGenerator.cs" />
-    <Compile Include="Workarounds\CodeDom\Compiler\CompilerError.cs" />
-    <Compile Include="Workarounds\CodeDom\Compiler\CompilerErrorCollection.cs" />
-    <Compile Include="Workarounds\CodeDom\Compiler\CompilerInfo.cs" />
-    <Compile Include="Workarounds\CodeDom\Compiler\CompilerParameters.cs" />
-    <Compile Include="Workarounds\CodeDom\Compiler\CompilerResults.cs" />
-    <Compile Include="Workarounds\CodeDom\Compiler\GeneratorSupport.cs" />
-    <Compile Include="Workarounds\CodeDom\Compiler\ICodeCompiler.cs" />
-    <Compile Include="Workarounds\CodeDom\Compiler\ICodeGenerator.cs" />
-    <Compile Include="Workarounds\CodeDom\Compiler\TempFiles.cs" />
-    <Compile Include="Workarounds\CodeDom\MemberAttributes.cs" />
-    <Compile Include="Workarounds\CodeDom\RedistVersionInfo.cs" />
-    <Compile Include="Workarounds\ConfigurationValidatorBase.cs" />
-    <Compile Include="Workarounds\DummyTypes\AppDomain.cs" />
-    <Compile Include="Workarounds\DummyTypes\ConfigurationElement.cs" />
-    <Compile Include="Workarounds\DummyTypes\ConfigurationElementCollection.cs" />
-    <Compile Include="Workarounds\DummyTypes\ConfigurationErrorsException.cs" />
-    <Compile Include="Workarounds\DummyTypes\ConfigurationException.cs" />
-    <Compile Include="Workarounds\DummyTypes\ConfigurationPropertyCollection.cs" />
-    <Compile Include="Workarounds\DummyTypes\ConfigurationSection.cs" />
-    <Compile Include="Workarounds\DummyTypes\ConfigurationSectionGroup.cs" />
-    <Compile Include="Workarounds\DummyTypes\ConnectionStringSettingsCollection.cs" />
-    <Compile Include="Workarounds\DummyTypes\Evidence.cs" />
-    <Compile Include="Workarounds\DummyTypes\PermissionSet.cs" />
-    <Compile Include="Workarounds\DummyTypes\SecurityManger.cs" />
     <Compile Include="Workarounds\LocalAppContextSwitches.cs" />
```

Comment:
What about these last 2? If they need to remain then they should not be in a "Workarounds" directory.